package/appserver/static/javascript/views/setup_page.js

```diff
@@ -35,7 +35,10 @@ export async function perform(splunk_js_sdk, setup_options) {
             application_name_space,
         );
 
-        let { clientId, clientSecret, index, serverUrl, startDate, ...properties } = setup_options;
+        let { clientId, clientSecret, index, rawServerUrl, startDate, ...properties } = setup_options;
+
+        // Parse server URL
+        const serverUrl = new URL(rawServerUrl);
```

Comment:
This is a regression, `URL` requires the protocol to be present, either `http:` or `https:` and throws exception when none is provided, where it was working fine before (we added it in python app backend when missing).
```suggestion
        const serverUrlContainsProtocol = rawServerUrl.startsWith("http://") || rawServerUrl.startsWith("https://");
        const serverUrl = new URL(serverUrlContainsProtocol ? rawServerUrl : "https://" + rawServerUrl);
```